recipe/meta.yaml

```diff
@@ -12,30 +12,21 @@ source:
   sha256: {{ checksum }}
 
 build:
-  number: 0
+  number: 1
+  # These are present when the new environment is created
+  # so we have to exempt them from the list of initial files
+  # for conda-build to realize they should be included.
   entry_points:
     - conda = conda.cli:main
     - conda-env = conda_env.cli.main:main
   always_include_files:
```

Comment:
This section from the `defaults` recipe is

```
  always_include_files:
    - bin/conda                 [unix]
    - bin/activate              [unix]
    - bin/deactivate            [unix]
    - Scripts/activate.bat      [win]
    - Scripts/deactivate.bat    [win]
    - Scripts/activate          [win]
    - Scripts/deactivate        [win]
```

That's correct and what should be used here.